packages/plugins/EVM/src/state/IdentityService.ts

```diff
@@ -284,9 +284,31 @@ export class IdentityService extends IdentityServiceState<ChainId> {
             this.getSocialAddressesFromMaskX(identity),
             this.getSocialAddressFromLens(identity),
         ])
-        const identities = allSettled
+        const identities_ = allSettled
             .flatMap((x) => (x.status === 'fulfilled' ? x.value : []))
             .filter(Boolean) as Array<SocialAddress<ChainId>>
-        return uniqBy(identities, (x) => [x.type, x.label, x.address.toLowerCase()].join('_'))
+
+        const identities = uniqBy(identities_, (x) => [x.type, x.label, x.address.toLowerCase()].join('_'))
+
+        if (!getNextIDPlatform() && !(process.env.channel === 'stable' && process.env.NODE_ENV === 'production')) {
+            return identities
+        }
+
+        const allSettledTwitterHandler = await Promise.allSettled(
+            identities.map(async (x) => {
+                const bindingProofs = await NextIDProof.queryProfilesByRelationService(x.address)
```

Comment:
The `identities` could have duplicate addresses.

Reply:
Can we query it like `queryProfilesByRelationService(uniqBy(identities.map(x => x.address), x => x.toLowerCase()))`?

Reply:
> The `identities` could have duplicate addresses.

fixed.


> Can we query it like this?
> 
> ```ts
> queryProfilesByRelationService(uniqBy(identities.map(x => x.address), x => x.toLowerCase()))
> ```

No, backend limitation.

Reply:
I see the requirement only mentioned `ENS` and `Lens`. So detecting `SocialAddressType` is also helpful. However, some of them might not need to verify by API.